ppr-api/src/ppr_api/models/__init__.py

```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 
 """This exports all of the models and schemas used by the application."""
-from .db import db  # noqa: I001
 from .address import Address
 from .client_code import ClientCode
 from .client_code_historical import ClientCodeHistorical
 from .court_order import CourtOrder
+from .db import db  # noqa: I001
```

Comment:
i've had issues with the db being imported in alphabetical order, as it's already imported by now in address, hence the 1001

Reply:
Update checked in with order change and disable of warning removed.